### PR TITLE
Issue#6 - Filter images by tag

### PR DESCRIPTION
**Pull request by Naman-sopho**

### CODE REVIEW

**Comment by Naman-sopho** [comment]
[Issue 6: Images by Tag](https://trello.com/c/MC6uw5Nx/413-issue-6-images-by-tag)

File: app/views/images/index.html.erb [hunk 1]
```diff
@@ -10,7 +10,7 @@
           <img class="p-4" src="<%= image.link %>"/>
           <div>
             <% image.tag_list.each do |tag| %>
-              <span> <%= tag %> </span>
+              <span> <%= link_to tag, images_path(tag: tag), class: 'js-tag' %> </span>
```
Comment (bg43179) [inline app/views/images/index.html.erb:13]: Do we use the `js-tag` elsewhere?

File: test/controllers/images_test.rb [hunk 2]
```diff
@@ -107,4 +105,35 @@ def test_index
...
+
+  def test_index__with_tags
+    Image.create(link: 'https://www.appfolio.com/_nuxt/img/markets-residential@2x_1080px-min.ae96531.png',
+                 tag_list: %w[tag01 tag02])
+    tags_list = %w[tag1 tag2]
+    image1 = Image.create(link: 'https://www.appfolio.com/_nuxt/img/product-communication-and-service_480px-min.4111e75.png',
+                          tag_list: tags_list)
+
+    get "/images?tag=#{tags_list[0]}"
+
```
Comment (bg43179) [inline test/controllers/images_test.rb:117]: I think we can change this to 
```ruby
image1 = Image.create(link: 'https://www.appfolio.com/_nuxt/img/product-communication-and-service_480px-min.4111e75.png', %w[tag1 tag2])

get "/images?tag=tag1"
```
All these variables with similar name make it a little bit hard to follow,

**Review by bg43179 — APPROVED** [review]
LGTM
